.github/workflows/main.yml

```diff
@@ -17,14 +18,27 @@ jobs:
     strategy:
       matrix:
         node-version: [10.x, 12.x,13.x]
+
     steps:
     - uses: actions/checkout@v2
     - name: Use Node.js ${{ matrix.node-version }}
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
+               
+    - name: Upload coverage to Codecov
```

Comment:
**Upload coverage to codecov**  should come after `yarn run build ` , `yarn test:unit --coverage`.

It wont upload anything

Reply:
Shift it